A/Aeron/build_tarballs.jl

```diff
@@ -15,46 +15,61 @@ script = raw"""
 apk update
 apk add openjdk11 hdrhistogram-c-dev libbsd-dev util-linux-dev
 cd $WORKSPACE/srcdir/aeron
-mkdir build && cd build
-CMAKE_FLAGS=(-DCMAKE_INSTALL_PREFIX=$prefix
--DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+mkdir -p /sys/devices/system/cpu/
+echo 1-${nproc} > /sys/devices/system/cpu/possible
+echo $prefix
+if [[ "${target}" == *linux* ]]; then
+    sed -i 's/check_symbol_exists(poll "poll.h" POLL_PROTOTYPE_EXISTS)/set(POLL_PROTOTYPE_EXISTS True)/g' aeron-driver/src/main/c/CMakeLists.txt
+    sed -i '1s;^;add_compile_options("-lrt")\nlink_libraries("-lrt")\n;' CMakeLists.txt
+    sed -i 's/check_type_size("struct mmsghdr" STRUCT_MMSGHDR_TYPE_EXISTS)/set(STRUCT_MMSGHDR_TYPE_EXISTS True)/g' aeron-driver/src/main/c/CMakeLists.txt
+fi
+sed -i 's/find_library(LIBBSD_EXISTS NAMES bsd libbsd)/set(LIBBSD_EXISTS False)/g' aeron-driver/src/main/c/CMakeLists.txt
+sed -i 's/find_library(LIBUUID_EXISTS NAMES uuid libuuid libuuid.dll)/set(LIBUUID_EXISTS False)/g' aeron-driver/src/main/c/CMakeLists.txt
+CMAKE_FLAGS=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
 -DCMAKE_BUILD_TYPE=Release
 -DCMAKE_C_EXTENSIONS=ON
--DBUILD_AERON_DRIVER=OFF
+-DBUILD_AERON_DRIVER=ON
 -DBUILD_AERON_ARCHIVE_API=OFF
 -DAERON_TESTS=OFF
 -DAERON_SYSTEM_TESTS=OFF
 -DAERON_BUILD_SAMPLES=OFF
 -DAERON_BUILD_DOCUMENTATION=OFF
 -DAERON_ENABLE_NONSTANDARD_OPTIMIZATIONS=OFF
 -DAERON_INSTALL_TARGETS=ON)
+mkdir build && cd build
+export LDFLAGS="-L${libdir}"
 cmake .. "${CMAKE_FLAGS[@]}"
-make -j${nproc}
+make -j${nproc} aeron_client aeron_driver_static
 make install
+cp /usr/local/lib/*aeron* ${libdir}
+mkdir -p ${bindir}
+cp /usr/local/bin/aeronmd ${bindir}
```

Comment:
Maybe there's a way to change the installation directory instead of manually moving files around afterwards.

Reply:
I spent some time on this without any luck. At least, I couldn't find a cleaner way then just moving them.

Reply:
What happened to the line `CMAKE_FLAGS=(-DCMAKE_INSTALL_PREFIX=$prefix` whose goal was exactly to set the destination prefix?

Reply:
It doesn't do what we want exactly, it places it in $prefix/lib and $prefix/binaries

Reply:
It places _what_ in where?

Reply:
I'll check through this again

Reply:
Ok fixed now. Thanks for your patience!